### PR TITLE
Adding home starting positions for simulated ur10 and hand E.

### PR DESCRIPTION
**Pull request by ethanfowler**
Optional, and defaults to false. No change to the current launch interface, therefore shouldn't break anything.

### CODE REVIEW

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 1]
```diff
@@ -74,7 +76,9 @@
     <!-- HAND -->
     <param name="/hand/mapping/$(arg hand_serial)" value="$(arg hand_id)"/>
     <param name="/hand/joint_prefix/$(arg hand_serial)" value="$(arg hand_id)_"/>
-    <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model $(arg robot_name)" respawn="false" output="screen"/>
+    <arg if="$(arg start_home)" name="spawn_model_extra_args" value="-unpause -J ra_shoulder_pan_joint 1.147 -J ra_elbow_joint 1.695 -J ra_wrist_1_joint -1.395 -J ra_wrist_2_joint -1.584 -J ra_shoulder_lift_joint -1.926 -J ra_wrist_3_joint 1.830" />
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: I think we should have an arg "home_position" with these values as default but clearly defined so we can change those if we want from the ur10, etc launch file

Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch]: Agreed - I'll make the changes

Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch]: Changes made!

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 2]
```diff
@@ -21,6 +21,11 @@
   <arg name="world" default="$(find sr_description_common)/worlds/sr_lab.world" if="$(arg scene)"/>
   <!-- Set the scene file -->
   <arg name="scene_file" default="$(find sr_description_common)/scenes/lab.scene"/>
+  <!-- Set to true if you want the robot to start in the "home" positions -->
+  <arg name="start_home" default="false"/>
+  <!-- Definition of "home" position, in joint-angle pairs -->
+  <arg name="home_angles" default="-J ra_shoulder_pan_joint 1.147 -J ra_elbow_joint 1.695 -J ra_wrist_1_joint -1.395 -J ra_wrist_2_joint -1.584 -J ra_shoulder_lift_joint -1.926 -J ra_wrist_3_joint 1.830"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch:27]: probably the -J is not needed here and can be added later

File: sr_robot_launch/launch/sr_right_ur10arm_hand.launch [hunk 3]
```diff
@@ -22,6 +22,8 @@
   <arg name="eth_port" default="$(optenv ETHERCAT_PORT eth0)"/>
   <!-- Set whether to load a scene to restrain the movement of the robot -->
   <arg name="scene" default="false"/>
+  <!-- Set to true if you want the robot to start in the "home" positions -->
+  <arg name="start_home" default="false"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_right_ur10arm_hand.launch:26]: we should add the home position values here so we know where to change them

File: sr_robot_launch/launch/sr_ur_arm_hand.launch (hunk 2 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch:27]: But there's a -J for every pair, which I can't think of any easy way around :/

File: sr_robot_launch/launch/sr_right_ur10arm_hand.launch (hunk 3 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_right_ur10arm_hand.launch:26]: Added!

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**